inc/dbmysql.class.php

```diff
@@ -1373,106 +1373,6 @@ function truncateOrDie($table, $message = '') {
       return $res;
    }
 
-   /**
```

Comment:
I'm note sure about this one, i must verify, but it may be used by advanced dahboards plugin